services/ui-src/package.json

```diff
@@ -29,6 +29,7 @@
     "react-bootstrap": "^2.0.1",
     "react-dom": "^17.0.1",
     "react-dropzone": "^11.4.2",
+    "react-error-boundary": "^3.1.4",
```

Comment:
The [react-error-boundary](https://www.npmjs.com/package/react-error-boundary) package implements React's concept of Error Boundaries as specified in the [official docs here](https://reactjs.org/docs/error-boundaries.html).

Reason: I wanted to avoid having to recreate this functionality with our own typings and unit tests. 
Justification: 3.9k stars, 2.4M weekly downloads, well-maintained, 100% test coverage. Owned by Brian Vaughn (React Dev Tools) and Kent Dodds (all the stuff).

Reply:
👍 Yeah. React really really needs add hook support for error boundaries. Until then, react-error-boundary is definitely the way.

Reply:
Yes PLEASE.